lib/read.js

```diff
@@ -95,6 +93,20 @@ class ReadSql extends AdapterRead {
         this.timeField = options.timeField ||
             (options.from || options.to || options.last ? 'time' : undefined);
     }
+    
+    addSorting() {
+        if (this.sortFields) {
+            // adds multiple sort orders
+            this.sortFields.forEach((sortObj) => {
```

Comment:
I would put this down by line 128 in `addOptimizations`, seems a little scattered to have it here.